tests_integration/__tests__/__snapshots__/patterns.js.snap

```diff
@@ -85,3 +95,21 @@ exports[`no errors on empty patterns (stderr) 1`] = `""`;
 exports[`no errors on empty patterns (stdout) 1`] = `""`;
 
 exports[`no errors on empty patterns (write) 1`] = `Array []`;
+
+exports[`should not expand directories (stderr) 1`] = `
+"[error] No matching files. Patterns tried: directory other-directory !**/node_modules/** !./node_modules/** !**/.{git,svn,hg}/** !./.{git,svn,hg}/**
+"
+`;
+
+exports[`should not expand directories (stdout) 1`] = `""`;
+
+exports[`should not expand directories (write) 1`] = `Array []`;
+
+exports[`should not support dot pattern (stderr) 1`] = `
+"[error] No matching files. Patterns tried: . !**/node_modules/** !./node_modules/** !**/.{git,svn,hg}/** !./.{git,svn,hg}/**
+"
```

Comment:
Quick explain: we don't support directory name, so `prettier directory other-directory` should throw